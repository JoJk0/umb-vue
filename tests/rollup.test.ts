// import path from 'node:path'
// import { testFixtures } from '@sxzz/test-utils'
import { describe } from 'vitest'
// import UnpluginStarter from '../src/rollup'

describe('rollup', async () => {
  // const { dirname } = import.meta
  // await testFixtures(
  //   '*.js',
  //   () => {
  //     // const { snapshot } = await rollupBuild(id, [UnpluginStarter()])
  //     // return snapshot
  //     return {}
  //   },
  //   { cwd: path.resolve(dirname, 'fixtures'), promise: true },
  // )
})
