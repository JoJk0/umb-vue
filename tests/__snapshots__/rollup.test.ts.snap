// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`rollup > basic.js 1`] = `{}`;
